### PR TITLE
Changes to README

### PR DESCRIPTION
**Pull request by jvtrigueros**
I've made several changes as I mentioned in Issue #3. One of the biggest changes that I've made was to support the latest SDK and make the VM include a WM to allow the user to be able to use the built in emulator.

In the `README.md`, I also mention that you can change to a different version of the SDK by checking out a tag. Those tags aren't included in this PR, it is a nice feature, maybe? I don't see many people going back to older versions of the SDK, so we can either remove that part or create tags for those sections.

Anyway, here are my changes =) I can squash them all if that makes life easier for you.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,39 +1,52 @@
...
-It's not pretty, so pull requests are happily taken.
+Clone this repository
 
-Use it at your own risk.
+    git clone https://github.com/shalecraig/vagrant-pebble
+    cd vagrant-pebble
 
-Instructions
-------------
+_Optional:_ Checkout the tag matching desired SDK version, `master` is on v3.0-dp7.
```
Comment (shalecraig) [inline README.md]: As of right now, I'm not entirely sure how much sense it makes to develop against old versions of Pebble library. Unless you can convince me that it's useful, can you remove the line about the tag?

File: sources/pebble_vm_provision_as_root.sh [hunk 2]
```diff
@@ -13,9 +12,10 @@ echo "                     Upgrading                     "
 echo "---------------------------------------------------"
 sudo DEBIAN_FRONTEND=noninteractive apt-get --assume-yes --option Dpkg::Options::="--force-confdef" --option Dpkg::Options::="--force-confold" upgrade
 echo "---------------------------------------------------"
-echo "                     Installing                    "
+echo "           Installing Dev Dependencies             "
 echo "---------------------------------------------------"
-sudo apt-get install --assume-yes git-core python-pip python2.7-dev language-pack-en-base
+sudo apt-get install --assume-yes git-core python-pip python2.7-dev language-pack-en-base \
```
Comment (shalecraig) [inline sources/pebble_vm_provision_as_root.sh]: If you're editing these anyways, can you make this one-per-line? it makes diffs easier to read.

If you don't get to it, I'll do it later.

**Comment by shalecraig** [comment]
Hey @jvtrigueros, thanks for your first pull-request on the vagrant-pebble repo!

I've commented in the diff containing some feedback based on a quick overview.

It's probably a good idea to see what @cpwc and @sebflipper feel about this before merging.

As I mentioned in #2, I don't have that much time to maintain this project anymore, so I gave you commit access to make any changes you'd like.

File: README.md (hunk 1 above)
Comment (cpwc) [inline README.md]: @shalecraig I guess some devs would like to develop against v2.9 instead of v3.0 since it still under dp?

**Comment by cpwc** [comment]
@shalecraig looks good :+1: 

@jvtrigueros thank you for your pull request. I am actually waiting for SDK to leave DP before updating the repo tho. But tagging is a great idea to pull specific SDK.

**Comment by jvtrigueros** [comment]
@shalecraig I'll make the changes on `pebble_vm_provision_as_root.sh`. 

Should I still remove the bit about checking out a different branch to use a different SDK version?

**Comment by shalecraig** [comment]
As soon as you make the changes, consider this :+1:'d.

@cpwc: feel free to accept the PR once you're happy.

**Comment by cpwc** [comment]
Thanks @shalecraig. Will do.

**Comment by jvtrigueros** [comment]
I've made the one change about keeping the `apt-get` dependencies in one line. It seems like it's a good idea to keep the tags until the SDK moves out of DP. If that's still the consensus then I'm done editing :)

File: sources/pebble_vm_provision_as_vagrant.sh [hunk 3]
```diff
@@ -8,23 +10,26 @@ echo "---------------------------------------------------"
...
 echo "---------------------------------------------------"
 mkdir -p /home/vagrant/pebble-dev
-chown vagrant:vagrant /home/vagrant/pebble-dev
 cd /home/vagrant/pebble-dev
 echo "---------------------------------------------------"
 echo "              Getting the pebble sdk               "
 echo "---------------------------------------------------"
-wget http://assets.getpebble.com.s3-website-us-east-1.amazonaws.com/sdk2/PebbleSDK-2.9.tar.gz
-tar -zxf PebbleSDK-2.9.tar.gz
+wget -q -a /vagrant/pebble.log \
```
Comment (cpwc) [inline sources/pebble_vm_provision_as_vagrant.sh]: keep this in one line too

File: sources/pebble_vm_provision_as_vagrant.sh [hunk 4]
```diff
@@ -8,23 +10,26 @@ echo "---------------------------------------------------"
...
-echo 'export PATH=/home/vagrant/pebble-dev/PebbleSDK-2.9/bin:$PATH' >> /home/vagrant/.profile
-source /home/vagrant/.profile
-cd /home/vagrant/pebble-dev/PebbleSDK-2.9
+echo "export PATH=/home/vagrant/pebble-dev/$PEBBLE_SDK/bin:\$PATH" >> /home/vagrant/.profile
+cd /home/vagrant/pebble-dev/$PEBBLE_SDK
 echo "---------------------------------------------------"
 echo "               Getting ubuntu tools                "
 echo "---------------------------------------------------"
-wget http://assets.getpebble.com.s3-website-us-east-1.amazonaws.com/sdk/arm-cs-tools-ubuntu-universal.tar.gz
+wget -q -a /vagrant/pebble.log \
```
Comment (cpwc) [inline sources/pebble_vm_provision_as_vagrant.sh]: keep this in one line too

**Comment by cpwc** [comment]
@jvtrigueros last 2 edits and we are done

**Comment by jvtrigueros** [comment]
Sorry I got super slammed at work, I'm working on the changes. I'll have them up tonight!

**Comment by jvtrigueros** [comment]
I have combined the other broken lines, but one of the ones you mentioned is in a commit not at the head. I'm afraid I'd have to rebase to make that happen, do you really want me to do that?

**Comment by cpwc** [comment]
@jvtrigueros I think we can make commit to correct that after the merge.

**Comment by jvtrigueros** [comment]
I didn't want to sound lazy it's just far back in history. I'll go ahead and do the rebase. It'll just affect all the commits on top. 



—

File: README.md [hunk 5]
```diff
@@ -1,39 +1,52 @@
...
+    Creating new project test
+    vagrant@vagrant-ubuntu-trusty-64:~$ cd test/
+    vagrant@vagrant-ubuntu-trusty-64:~/test$ pebble build
+    ...
+    'build' finished successfully (0.160s)
+
+## Using Emulator
+
+To use the emulator, you'll need to start VirtualBox in GUI mode. You can do this by
+uncommenting [Vagrantfile:10](https://github.com/jvtrigueros/vagrant-pebble/blob/v3.0-dp7/Vagrantfile#L10).
```
Comment (shalecraig) [inline README.md]: Can you remove the link, or just inline what the modification should be?

Something like the below should be fine.

> uncommenting this line in the Vagrantfile:
> 
> `vb.gui = true`

**Comment by shalecraig** [comment]
On second thought, this sounds good to merge - I'll just modify the line my comment was on.

Thanks for dragging this PR to the finish line, @jvtrigueros!

**Comment by jvtrigueros** [comment]
Thanks @shalecraig, I'll try to be a good contributing member of this repo =)

**Pull request merged**